### ISSUE TITLE
unknown_error:Database encountered an unknown error

### ISSUE THREAD

**Issue by gerroon** [vrtmrz/obsidian-livesync#347]
Hi

I am hosting the sync server in Docker/

` Fauxton on  [Apache CouchDB](http://couchdb.apache.org/) v. 3.3.3` What says in the web ui

I have been syncing fine for a while but suddenly the sync stopped working, and I now see it throws some kind of error and I am not sure what the issue is.

When I check database connection on both desktop and Ios, it says connected.   I do not use encryption.

```
12/9/2023, 6:10:09 PM->Check or pull from db:Store/assets/Untitled_Artwork+40.png OK
12/9/2023, 6:10:09 PM->Check or pull from db:Store/assets/latest.3.jpeg
12/9/2023, 6:10:09 PM->Check or pull from db:Store/assets/latest.5.jpeg
12/9/2023, 6:10:10 PM->Check or pull from db:Store/assets/affinity-publisher-2-dark-landscape-090520190839.svg OK
12/9/2023, 6:10:10 PM->Check or pull from db:Store/assets/eBV9PwY2Kxd6NsQjVqtzZ.jpeg OK
12/9/2023, 6:10:10 PM->Check or pull from db:Store/assets/latest.png
12/9/2023, 6:10:10 PM->Check or pull from db:Store/assets/image-1.png OK
12/9/2023, 6:10:10 PM->Check or pull from db:Store/assets/image.png OK
12/9/2023, 6:10:10 PM->Check or pull from db:Store/assets/latest.1.png OK
12/9/2023, 6:10:10 PM->Check or pull from db:Store/assets/mind-map-obsidian.jpeg
12/9/2023, 6:10:10 PM->Check or pull from db:Store/assets/settings (1).png
12/9/2023, 6:10:10 PM->Check or pull from db:Store/assets/image-2.png OK
12/9/2023, 6:10:10 PM->Check or pull from db:Store/assets/settings.png
12/9/2023, 6:10:10 PM->Check or pull from db:Store/assets/z06mp0up14glqidybtlo.png
12/9/2023, 6:10:10 PM->Check or pull from db:Store/assets/latest.2.png OK
12/9/2023, 6:10:10 PM->Check or pull from db:Store/assets/latest.3.jpeg OK
12/9/2023, 6:10:10 PM->Check or pull from db:Store/assets/latest.5.jpeg OK
12/9/2023, 6:10:10 PM->Check or pull from db:Store/assets/latest.png OK
12/9/2023, 6:10:10 PM->Check or pull from db:Store/assets/mind-map-obsidian.jpeg OK
12/9/2023, 6:10:10 PM->Check or pull from db:Store/assets/settings (1).png OK
12/9/2023, 6:10:10 PM->Check or pull from db:Store/assets/settings.png OK
12/9/2023, 6:10:10 PM->Check or pull from db:Store/assets/z06mp0up14glqidybtlo.png OK
12/9/2023, 6:10:10 PM->UPDATE STORAGE: PASS:269, FAILED:0
12/9/2023, 6:10:10 PM->CHECK FILE STATUS:2/2
12/9/2023, 6:10:10 PM->CHECK FILE STATUS:2/2: PASS:54, FAILED:0
12/9/2023, 6:10:10 PM->CHECK FILE STATUS:1/2
12/9/2023, 6:10:10 PM->STORAGE -> DB :note/note.md
12/9/2023, 6:10:10 PM->1702166789 > 1701825113
12/9/2023, 6:10:10 PM->Chunk save failed:
12/9/2023, 6:10:10 PM->unknown_error:Database encountered an unknown error
12/9/2023, 6:10:10 PM->note could not saved:note/note.md
12/9/2023, 6:10:10 PM->DB <- STORAGE (plain) note/note.md
12/9/2023, 6:10:10 PM->CHECK FILE STATUS:1/2: PASS:100, FAILED:0
12/9/2023, 6:10:10 PM->Initialized, NOW TRACKING!
12/9/2023, 6:10:10 PM->Content saved:Store/__Bookmarks__.md_ ,chunks: 1 (new:0, skip:1, cache:0)
12/9/2023, 6:10:10 PM->Error while loading Self-hosted LiveSync

```

**Comment by dlphnkck** [comment]
Hi. Sorry to bother you from the side.
Did you find the following article helpful?

[Replication error: unknown\_error:Database encountered an unknown error · Issue \#257 · vrtmrz/obsidian\-livesync](https://github.com/vrtmrz/obsidian-livesync/issues/257)

If you look at the error log, for example

```log
QuotaExceededError:Failed to PutOrAdd in database because not enough space for domain
```
```log
the size of xx.png is 300MB, it's a big image file, and I sometimes can get OOM error while opening Obsidian.
```

Have you seen any error messages that say?

Or, like [QuotaExceededError in ios · Issue #254 · vrtmrz/obsidian-livesync](https://github.com/vrtmrz/obsidian-livesync/issues/254), the database has run out?

That's all.
My apologies if this is misplaced.

**Comment by gerroon** [comment]
I did not see such error, given that cant' be the issue because I have a lot of space on  the clients and the server. I mainly checked the logs on the desktop client since it is easier to take a look at . I will try to look at it on the mobile client as well. 

I actually went ahead and started a new database and syncs to make sure. I will follow up in this bug if I get the same issue with the new database.
